compiler/ast.nim

```diff
@@ -310,7 +310,9 @@ type
                       #
                       # This is disallowed but can cause the typechecking to go into
                       # an infinite loop, this flag is used as a sentinel to stop it.
-
+    sfKeep            # A flag for types that indicates the compiler to emit the type
+                      # even if it is not used. 
+                      
```

Comment:
Why not instead use `sfExportc`?

Reply:
Because of what you said here https://forum.nim-lang.org/t/9305
Im ok with changing it, but it does more things. Doesnt it? For instance, we would export everything, even if not used, at least for the headers. 

BTW can you give me some directions on the header question on discord?

Reply:
Please use `sfExportc`, we don't need yet another pragma.